src/components/Card/Card.scss

```diff
@@ -0,0 +1,16 @@
+.card {
```

Comment:
Also just try to following the naming convention, each team might be different, but we'll stick with lower case for file names